common/src/main/java/com/wynntils/wynn/model/scoreboard/quests/QuestHandler.java

```diff
@@ -53,5 +55,6 @@ public void onSegmentRemove(Segment segment, ScoreboardModel.SegmentType segment
 
     public static void resetCurrentQuest() {
         currentQuest = null;
```

Comment:
Maybe move extract both these changes to a common private `setQuest` helper? To help make sure we send the event properly if we ever will set it from new code.